packages/cli/src/commands/start.ts

```diff
@@ -232,6 +232,12 @@ export class Start extends BaseCommand {
 
 		await Container.get(OrchestrationHandlerMainService).init();
 
+		/**
+		 * @tech_debt This is temporarily necessary so that worker view is accessible
+		 * in multi-main mode.
+		 */
+		await Container.get(SingleMainSetup).init();
+
 		const multiMainSetup = Container.get(MultiMainSetup);
```

Comment:
```suggestion
		const multiMainSetup = Container.get(MultiMainSetup);
		Container.set(SingleMainSetup, multiMainSetup);
```

Is there a chance we would duplicate things? Such as duplicating messages sent or broadcasting stuff?

Also is it possible instead of initializing both `SingleMainSetup` and `MultiMainSetup` to simply initialize Multi Main and feed it to the Container? I don't know if the above would work properly, but I guess it's something we could try wdyt?